client-proxy/validator/validator.go

```diff
@@ -0,0 +1,16 @@
+package validator
+
+type Validator struct{}
+
+type IEventValidator interface {
+	Validate(event string) error
+}
+
+func (Validator *Validator) Validate(event string) (err error) {
+	// Not yet implemented.
```

Comment:
What sort of validation on the event will be applied?

Reply:
Validation against the open lineage event json schema.

Reply:
That the event is valid schema? or a valid event based on the OL spec?

Reply:
If the event is valid based on the spec.